typo: web/resource-manager/Microsoft.Web

nschonni opened the pull request:
- API capitilization
- policies -> polcies
- accesible -> accessible
- conenction -> connection
- Contsent -> Consent
- Retrive -> Retrieve
- specificed -> specified
- subcription -> subscription
- multiRool -> multiRole
- Sevice -> Service
- VNets capitalization
- configuraiton -> configuration
- performa -> perform
- BizTalk capitalization
- ceritificate -> certificate
- expriration -> expiration
- Double word "be"
- crendentials -> credentials
- serverfarm -> server farm
- polymophic -> polymorphic
- Double word "to"
- resourse -> resource
- defintions -> definitions
- Succesfully -> Successfully
- Instrunctions -> Instructions
- corelated -> correlated
- Reponse -> Response
- recommedation -> recommendation
- availbility -> availability
- VM's -> VMs
- desciption -> description
- identifider -> identifier
- siteextensions -> site extensions
- sourcecontrol -> source control
- htere -> there
- sku's -> SKUs
- shoud -> should

AutorestCI commented:
# Automation for azure-sdk-for-ruby

azuresdkci commented:
Can one of the admins verify this patch?

AutorestCI commented:
# Automation for azure-sdk-for-js

AutorestCI commented:
# Automation for azure-sdk-for-python

AutorestCI commented:
# Automation for azure-sdk-for-go

AutorestCI commented:
# Automation for azure-sdk-for-java

AutorestCI commented:
# Automation for azure-sdk-for-node

jianghaolu commented on specification/web/resource-manager/Microsoft.Web/stable/2015-08-01/service.json:
```diff
@@ -5904,7 +5904,7 @@
         ],
         "summary": "List of metrics that can be queried for an App Service Plan",
-        "operationId": "ServerFarms_GetServerFarmMetricDefintions",
+        "operationId": "ServerFarms_GetServerFarmMetricDefinitions",
```
I understand this is to fix a typo - but existing SDKs will be broken - you might have to wait to fix this in the next API version.

nschonni replied:
Didn't notice the breaking change. I'll remove it from this PR

nschonni committed:
typo: web/resource-manager/Microsoft.Web

- API capitilization
- policies -> polcies
- accesible -> accessible
- conenction -> connection
- Contsent -> Consent
- Retrive -> Retrieve
- specificed -> specified
- subcription -> subscription
- multiRool -> multiRole
- Sevice -> Service
- VNets capitalization
- configuraiton -> configuration
- performa -> perform
- BizTalk capitalization
- ceritificate -> certificate
- expriration -> expiration
- Double word "be"
- crendentials -> credentials
- serverfarm -> server farm
- polymophic -> polymorphic
- Double word "to"
- resourse -> resource
- defintions -> definitions
- Succesfully -> Successfully
- Instrunctions -> Instructions
- corelated -> correlated
- Reponse -> Response
- recommedation -> recommendation
- availbility -> availability
- VM's -> VMs
- desciption -> description
- identifider -> identifier
- siteextensions -> site extensions
- sourcecontrol -> source control
- htere -> there
- sku's -> SKUs
- shoud -> should

nschonni commented:
@jianghaolu I pulled out the breaking changes and added the rest of the files in the namespace. I filed separate issues for the typos in properties and operation names.

nschonni commented:
@jianghaolu @naveedaz is there anything else to change for this?
